src/js/shape_editor/shape_manager.js

```diff
@@ -46,6 +46,7 @@ var ShapeManager = function ShapeManager(elementId, width, height, options) {
   this._orig_width = width;
   this._orig_height = height;
   this._zoom = 100;
+  this._shapeScalingFactor = 100;
```

Comment:
This might be a good place to add some docstring comments on what `_shapeScalingFactor` and `_zoom` are. When I've looked at this code recently, these concepts can get confusing. You could even include a link to this PR.